grails-app/services/com/unifina/service/UserService.groovy

```diff
@@ -69,6 +73,17 @@ class UserService {
 			// Transfer permissions that were attached to sign-up invitation before user existed
 			permissionService.transferInvitePermissionsTo(user)
 		}
+
+		List<Canvas> canvasExamples = Canvas.createCriteria().list {
```

Comment:
One afterthought: should we wrap these in `try-catch` to make 100% sure that new user creation doesn't fail if there are any exceptions thrown while copying the examples? Maybe there should be a `try-catch` for canvases and streams separately, so that one of them can succeed if one fails.